frontend/http/parser.go

```diff
@@ -47,43 +65,55 @@ func ParseAnnounce(r *http.Request, realIPHeader string, allowIPSpoofing bool) (
 	}
 	request.Peer.ID = bittorrent.PeerIDFromString(peerID)
 
+	// Determine the number of remaining pieces for the client.
```

Comment:
All of these (left, downloaded, uploaded) are numbers of bytes, not pieces, iirc.

Sorry, I should've noticed with the last review.